tools/regtesting/do_regtest

```diff
@@ -528,6 +529,7 @@ function run_regtest_dir() {
      fi
 
      printf "    %-50s %25s %20s %s\n" "${input_file}" "${test_value}" "${this_test}" "${leak_marker}" >> ${dir_out}/status/REGTEST_TASK_TESTS-$task
+     printf "%7.2f  %s\n" "${timing}" "${dir}/${input_file}" >> ${timings}
```

Comment:
please use `"${timings}"` to avoid problems should `${dir_out}` contain spaces or other strange characters

Reply:
Done. Thanks!